src/lib.rs

```diff
@@ -1,4 +1,4 @@
-#![feature(nll)]  // QUESTION: Do we still need this?
+// #![feature(nll)]  // QUESTION: Do we still need this?
```

Comment:
I don't know. But this line makes build error on my environment (Ubuntu 19.10), so comment out.